test-app/tests/unit/services/browser/window-test.ts

```diff
@@ -85,29 +85,6 @@ module('Service | browser/window', function (hooks) {
         assert.strictEqual(service.parent.location.href, loginPath);
       });
     });
-
-    module('Stubbing location.origin', function (hooks) {
-      setupBrowserFakes(hooks, {
-        window: {
-          location: { origin: 'http://init.ial', href: 'http://init.ial' },
```

Comment:
Dropping this test can be seen as a breaking change, see the [changelog comment](https://github.com/CrowdStrike/ember-browser-services/pull/399/files#diff-b9cc623d4d04c076a2cadea0ace8062484e9e0896e98f6de7f8e79a04ac10b0cR7)

@NullVoxPopuli any concerns?

Reply:
nope, the migration path seems straight-forward!